### PR TITLE
Update modules

### PR DESCRIPTION
**Pull request by christopher-mohr**
This updates the following nf-core modules
- multiqc
- gunzip
- dumpsoftwareversions

and adapts `epitopeprediction.nf` to the changes in the `multiqc` module.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/epitopeprediction/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/epitopeprediction _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: workflows/epitopeprediction.nf [hunk 1]
```diff
@@ -22,7 +22,7 @@ if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input sample
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-ch_multiqc_config        = file("$projectDir/assets/multiqc_config.yml", checkIfExists: true)
+ch_multiqc_config        = [file("$projectDir/assets/multiqc_config.yml", checkIfExists: true), []]
 ch_multiqc_custom_config = params.multiqc_config ? Channel.fromPath(params.multiqc_config) : Channel.empty()
```
Comment (ggabernet) [inline workflows/epitopeprediction.nf:26]:
```suggestion
ch_multiqc_custom_config = params.multiqc_config ? Channel.fromPath(params.multiqc_config) : Channel.empty()
ch_configs_mqc = ch_multiqc_config.mix(ch_multiqc_custom_config).ifEmpty([])
```

File: workflows/epitopeprediction.nf [hunk 2]
```diff
@@ -444,13 +444,12 @@ workflow EPITOPEPREDICTION {
...
 
     ch_multiqc_files = Channel.empty()
-    ch_multiqc_files = ch_multiqc_files.mix( Channel.from( ch_multiqc_config ) )
     ch_multiqc_files = ch_multiqc_files.mix( ch_multiqc_custom_config.collect().ifEmpty([]) )
     ch_multiqc_files = ch_multiqc_files.mix( CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect() )
     ch_multiqc_files = ch_multiqc_files.mix( ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml') )
 
     MULTIQC (
-        ch_multiqc_files.collect()
+        ch_multiqc_files.collect(), ch_multiqc_config
```
Comment (ggabernet) [inline workflows/epitopeprediction.nf:452]:
```suggestion
        ch_multiqc_files.collect(), ch_configs_mqc.collect()
```

Comment (ggabernet) [reply workflows/epitopeprediction.nf:452]: here the custom config should be mixed with the default config

File: workflows/epitopeprediction.nf [hunk 3]
```diff
@@ -444,13 +444,12 @@ workflow EPITOPEPREDICTION {
     ch_workflow_summary = Channel.value( workflow_summary )
 
     ch_multiqc_files = Channel.empty()
-    ch_multiqc_files = ch_multiqc_files.mix( Channel.from( ch_multiqc_config ) )
     ch_multiqc_files = ch_multiqc_files.mix( ch_multiqc_custom_config.collect().ifEmpty([]) )
```
Comment (ggabernet) [inline workflows/epitopeprediction.nf:447]:
```suggestion
```

**Review by ggabernet — COMMENTED** [review]
just some comments on the mqc configs, otherwise looks good

**Comment by ggabernet** [comment]
@apeltzer you were a bit trigger happy 🔫  😄 The tests were failing... 😆

**Comment by apeltzer** [comment]
Classic me 😂

**Review by ggabernet — APPROVED** [review]
LGTM now :)

**Pull request merged**